sdks/java/core/src/test/java/org/apache/beam/sdk/util/ApiSurfaceTest.java

```diff
@@ -76,7 +68,33 @@ public void testOurApiSurface() throws Exception {
   }
 
   private boolean classIsAllowed(Class<?> clazz) {
-    return  clazz.getName().startsWith("org.apache.beam");
+    return clazz.getName().startsWith("org.apache.beam")
```

Comment:
To allow only selected classes from a package, list them individually and don't include a prefix whitelist. When this list started I had in mind that there would be more of that, and less of these blanket allowances, hence I didn't make it just a list of allows packages. But since today it is, I will switch it over.